h2gis-functions/src/test/java/org/h2gis/functions/spatial/ogc/OGCSpatialFunctionTest.java

```diff
@@ -632,4 +632,42 @@ public void test_ST_LengthOnPolygon() throws SQLException {
         assertFalse(rs.next());
         rs.close();
     }
+
+    @Test
+    public void test_ST_DistanceSpherePointToPoint() throws Exception {
+        Statement st = connection.createStatement();
+        ResultSet rs = st.executeQuery("SELECT ST_DistanceSphere(buildings.position, bridges.position) " +
```

Comment:
Could you please move the tests into SpatialFunction2Test.java because ST_DistanceSphere is not an OGC spatial function.